### PR TITLE
Exempt Mojang from the privacy module

### LINKED ISSUE #497: Exempt Mojang employees from the Privacy module
## The Problem
When a mojang employee makes a comment that contains an email address (as far as Arisa knows), the comment gets unnecessarily restricted, quite like here:
https://bugs.mojang.com/browse/MCPE-105882?focusedCommentId=858817&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-858817

## Possible Solution
We should be trusting the owner of the platform not to post privacy-sensitive information, so mojang employees should be exempted from the check. Alternatively, the other way around, the module could be configured to specifically function only on e.g. the user, helper and moderator groups. That way no exception-logic is needed.

### PR DESCRIPTION
**Pull request by chandler05**
## Purpose
Fix #497 
#### Checklist
- [ ] Included tests
- [ ] Updated documentation in [README](https://github.com/mojira/arisa-kt/blob/master/README.md) and [wiki](https://github.com/mojira/arisa-kt/wiki)
- [ ] Tested in MCTEST-xxx

### CODE REVIEW

File: src/main/kotlin/io/github/mojira/arisa/modules/PrivacyModule.kt [hunk 1]
```diff
@@ -43,6 +43,7 @@ class PrivacyModule(
                 .filter { it.created.isAfter(lastRun) }
                 .filter { it.visibilityType == null }
                 .filter { it.body?.matches(patterns) ?: false }
+                .filterNot { it.getAuthorGroups()?.any { it == "staff" } ?: false }
```
Comment (SPGoding) [inline src/main/kotlin/io/github/mojira/arisa/modules/PrivacyModule.kt:46]:
```suggestion
                .filterNot { it.getAuthorGroups()?.any { it == "helper" || it == "staff" } ?: false }
```

**Review by SPGoding — APPROVED** [review]

File: src/main/kotlin/io/github/mojira/arisa/modules/PrivacyModule.kt (hunk 1 above)
Comment (chandler05) [reply src/main/kotlin/io/github/mojira/arisa/modules/PrivacyModule.kt:46]: I thought only Mojang employees were going to be exempt from this?

Comment (urielsalis) [reply src/main/kotlin/io/github/mojira/arisa/modules/PrivacyModule.kt:46]: All helpers and mods should be too

**Review by urielsalis — COMMENTED** [review]
.

**Pull request merged**